app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -35,40 +39,48 @@ public LoginActivitySteps(LoginActivity activityClass) {
     }
 
     @Given("^I have a LoginActivity")
-    public void I_have_a_LoginActivity() {
+    public void iHaveALoginActivity() {
         assertNotNull(getActivity());
     }
 
-    @When("^I input email (\\S+)$")
-    public void I_input_email(final String email) {
-        onView(withId(R.id.email)).perform(typeText(email));
-    }
-
     @When("^I input password \"(.*?)\"$")
-    public void I_input_password(final String password) {
+    public void iInputPassword(final String password) {
         onView(withId(R.id.password)).perform(typeText(password));
     }
 
     @When("^I press submit button$")
-    public void I_press_submit_button() {
+    public void iPressSubmit() {
         onView(withId(R.id.submit)).perform(scrollTo()).perform(click());
     }
 
-    @Then("^I should see error on the (\\S+)$")
-    public void I_should_see_error_on_the_editTextView(final String viewName) {
-        int viewId = (viewName.equals("email")) ? R.id.email : R.id.password;
-        int messageId = (viewName.equals("email")) ? R.string.msg_email_error : R.string.msg_password_error;
+    @Then("^I (true|false) continue to the MainActivity$")
+    public void iShouldContinueToNextActivity(boolean continuesToNextActivity) {
+        if (continuesToNextActivity) {
+            assertEquals(getCurrentActivity().getClass(), MainActivity.class);
+        } else {
+            assertEquals(getCurrentActivity().getClass(), LoginActivity.class);
+        }
+    }
 
-        onView(withId(viewId)).check(matches(hasErrorText(getActivity().getString(messageId))));
+    @Then("^I should see an auth error$")
+    public void iShouldSeeAuthError() {
+        onView(withId(R.id.error)).check(matches(isDisplayed()));
     }
 
-    @Then("^I should (true|false) auth error$")
-    public void I_should_see_auth_error(boolean shouldSeeError) {
-        if (shouldSeeError) {
-            onView(withId(R.id.error)).check(matches(isDisplayed()));
-        } else {
-            onView(withId(R.id.error)).check(matches(not(isDisplayed())));
+    Activity getCurrentActivity() {
```

Comment:
Can you make this a `private` method please. Also, I'm not quite sure how this is supposed to work... Does [line 43](https://github.com/ericcornelissen/NervousFish/blob/96acc8e91c2b4a817f4d83a09a84eecddba4b7e8/app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java#L43) reference this method?